main.py

```diff
@@ -1,58 +1,65 @@
 import requests
 import jmespath
 from bs4 import BeautifulSoup
-import re
-
-final_results = []
-titles = []
-posters = []
-rates = []
-genres = []
-directors = []
-results = {}
-
-response = requests.get('https://www.filmweb.pl/serials/search?endRate=10&orderBy=popularity&descending=true&startRate=9')
-soup = BeautifulSoup(response.content, 'html.parser')
-results_list = soup.find(class_="resultsList hits")
-
-results_list_posters = results_list.find_all(class_='poster__image')
-for poster in results_list_posters:
-    posters.append(jmespath.search('content', poster))
-
-results_list_titles = results_list.find_all(class_='filmPreview__title')
-for title in results_list_titles:
-    titles.append(title.get_text())
-
-results_list_rates = results_list.find_all(class_="rateBox__rate")
-for rate in results_list_rates:
-    rates.append(rate.get_text())
-
-for genre in results_list:
-    genre1 = genre.find(class_="filmPreview__info filmPreview__info--genres")
+
+HTTP_SUCCES = 200
+
+def find_movies():
+    response = requests.get('https://www.filmweb.pl/serials/search?endRate=10&orderBy=popularity&descending=true&startRate=9')
```

Comment:
To możesz wyrzucić do czegoś w stylu MOVIE_URL i umieść tak jak HTTP_SUCCESS gdzieś poza funckja.